catalog/dags/database/batched_update/batched_update_dag.py

```diff
@@ -253,6 +237,12 @@ def batched_update():
         query_id="{{ params.query_id }}",
     )
 
+    # Clean up the variable we used for tracking the last row
+    drop_variable = BashOperator(
+        task_id="drop_temp_airflow_variables",
+        bash_command=f"airflow variables delete {BATCH_START_VAR}",
+    )
```

Comment:
Would it make sense to potentially use [`Variable.delete`](https://airflow.apache.org/docs/apache-airflow/stable/_api/airflow/models/variable/index.html#airflow.models.variable.Variable.delete) here rather than a `BashOperator`? I guess I worry about environment/access issues that might appear on the CLI which we know wouldn't occur within Python 😅 No strong opinion here, just curious!

Reply:
Done!